server/methods/core/cart.js

```diff
@@ -840,7 +847,7 @@ Meteor.methods({
       }
     }
     if (!updated) {
-      // if we didn't do one of the inline updates, then run the update here
+      // if we didn"t do one of the inline updates, then run the update here
```

Comment:
Why were these replaced?

Reply:
I will fix and revert that now. Thanks for pointing that out